enterprise/server/cmd/executor/executor.go

```diff
@@ -100,26 +102,45 @@ func InitializeCacheClientsOrDie(cacheTarget string, realEnv *real_environment.R
 		}
 		log.Infof("Connecting to cache target: %s", cacheTarget)
 	}
+	addHealthCheck(conn, false /*=required*/, "grpc_cache_connection", realEnv.GetHealthChecker())
```

Comment:
did you mean to make this "not required"?

Reply:
Yeah, this is trying to replicate the current grpc_cache_connection health checker that reports healthiness if the connection goes idle: https://github.com/buildbuddy-io/buildbuddy/blob/a8e2ae9bed50644867d1a9475a98f0621dc77754/enterprise/server/cmd/executor/executor.go#L104-L117